stories/rating-indicator/rating-indicator.stories.js

```diff
@@ -0,0 +1,537 @@
+                
+                <input aria-label="5 star" type="radio" class="fd-rating-indicator__input" id="rating-max-value-7-5" name="rating-max-value-7" value="5">
+                <label class="fd-rating-indicator__label" for="rating-max-value-7-5"></label>
+                
+                <input aria-label="6 star" type="radio" class="fd-rating-indicator__input" id="rating-max-value-7-6" name="rating-max-value-7" value="6">
+                <label class="fd-rating-indicator__label" for="rating-max-value-7-6"></label>
+                
+                <input aria-label="7 star" type="radio" class="fd-rating-indicator__input" id="rating-max-value-7-7" name="rating-max-value-7" value="7">
+                <label class="fd-rating-indicator__label" for="rating-max-value-7-7"></label>
+            </div>
+            <span class="fd-rating-indicator__dynamic-text">(2 of 7)</span>
+        </div>
+    </div>
+`;
+
+DifferentValues.storyName = 'Different values';
+DifferentValues.parameters = {
+    docs: {
+        storyDescription: `
+It is possible to display the rating indicator with a different number of values. 
+It is highly recommended to use 5 as the maximum value, however you can go up to 7 
+if it is ideal for your use case.
+`
+    }
+};
+
+export const RTL = () => `
+    <div class="example-container" dir="rtl">
+        <div class="fd-rating-indicator">
+            <div class="fd-rating-indicator__container" aria-label="Star Rating (out of 5)">
+                <input aria-label="1 star" type="radio" class="fd-rating-indicator__input" id="rating-rtl-1" name="rating-rtl" value="1">  
+                <label class="fd-rating-indicator__label" for="rating-rtl-1"></label>
+                
+                <input aria-label="2 star" type="radio" class="fd-rating-indicator__input" id="rating-rtl-2" name="rating-rtl" value="2" checked>
+                <label class="fd-rating-indicator__label" for="rating-rtl-2"></label>
+                
+                <input aria-label="3 star" type="radio" class="fd-rating-indicator__input" id="rating-rtl-3" name="rating-rtl" value="3">
+                <label class="fd-rating-indicator__label" for="rating-rtl-3"></label>
+                
+                <input aria-label="4 star" type="radio" class="fd-rating-indicator__input" id="rating-rtl-4" name="rating-rtl" value=4">
+                <label class="fd-rating-indicator__label" for="rating-rtl-4"></label>
+                
+                <input aria-label="5 star" type="radio" class="fd-rating-indicator__input" id="rating-rtl-5" name="rating-rtl" value="5">
+                <label class="fd-rating-indicator__label" for="rating-rtl-5"></label>
+            </div>
+            <span class="fd-rating-indicator__dynamic-text">(2 من 5)</span>
+        </div>
+    </div>
+`;
+
+RTL.storyName = 'RTL';
+RTL.parameters = {
+    docs: {
+        storyDescription: 'The rating indicator can be displayed from right to left on the screen.'
```

Comment:
Please add this to the text:

The rating indicator can be displayed from right to left on the screen so that it may be used internationally.

Reply:
fixed